tests/callbacks/test_callbacks.py

```diff
@@ -4,118 +4,117 @@
 
 import tests.base.utils as tutils
 from pytorch_lightning import Callback
-from pytorch_lightning import Trainer, LightningModule
+from pytorch_lightning import Trainer, Evaluator, LightningModule
 from pytorch_lightning.callbacks import EarlyStopping, ModelCheckpoint
 from pytorch_lightning.loggers import TensorBoardLogger
 from tests.base import EvalModelTemplate
 
 
+class TestCallback(Callback):
```

Comment:
Will we split callbacks as well or will we have a single class that has to be attached to both, Trainer and Evaluator?

Reply:
At the moment it's a single class that has to be attached to both. Having evaluator specific callbacks in the future might make sense though.